main/main.c

```diff
@@ -611,6 +611,7 @@ PHP_INI_BEGIN()
 #ifdef PHP_WIN32
 	STD_PHP_INI_BOOLEAN("windows.show_crt_warning",		"0",		PHP_INI_ALL,		OnUpdateBool,			windows_show_crt_warning,			php_core_globals,	core_globals)
 #endif
+	STD_PHP_INI_ENTRY("syslog.highbit",			"0",		 PHP_INI_SYSTEM,	OnUpdateBool,		syslog_highbit,		php_core_globals, 		core_globals)
```

Comment:
Because it's a purely theoretical concern with a very, very tenuous connection to security? If you want to go into pointless hyptotheticals, why not consider EBCDIC systems where LF is not even mapped to the same code point as ASCII? If the spec does not specify an encoding, we have to make reasonable assumptions. The reasonable assumptions are a) forbid ASCII control chars and b) handle ASCII LF.

IMHO even having an ini option is completely unnecessary, but it we do add one, it should certainly allow non-ASCII codepoints by default.